ui/k-toolbar/main.kytos

```diff
@@ -232,6 +237,7 @@ export default {
         tag_value_z: "",
         service_level: "",
         sb_priority: "",
+        max_paths: "2",
```

Comment:
With the default "2", although it's correct, in the UI it becomes difficult to understand what the 2 means:

![20250410_104437](https://github.com/user-attachments/assets/1cddef36-5d5b-4788-a790-0ff8df9e780b)

Since with the value it doesn't show that's related to max paths. 

I'd recommend leaving this as empty and following a similar pattern that we have with other attrs like `service_level` https://github.com/kytos-ng/mef_eline/blob/master/ui/k-toolbar/main.kytos#L518, and not set in the request if it's not defined, and let the back-end set the correct default. Easier to maintain too. See what I mean?

Reply:
Leaving it empty and sending 2 if it is empty.